editors/sc-ide/CMakeLists.txt

```diff
@@ -154,6 +154,7 @@ set ( ide_src
     ${CMAKE_SOURCE_DIR}/QtCollider/widgets/web_page.cpp
 
     ${CMAKE_SOURCE_DIR}/common/SC_TextUtils.cpp
+    ${CMAKE_SOURCE_DIR}/common/localsocket_utils.cpp
```

Comment:
please also add the header, so that they will be included in the generated project files